artemis-hawtio/artemis-plugin/src/main/webapp/plugin/html/deleteDivert.html

```diff
@@ -0,0 +1,24 @@
+<!--
+  Licensed to the Apache Software Foundation (ASF) under one or more
+  contributor license agreements.  See the NOTICE file distributed with
+  this work for additional information regarding copyright ownership.
+  The ASF licenses this file to You under the Apache License, Version 2.0
+  (the "License"); you may not use this file except in compliance with
+  the License.  You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+  Architecture
+-->
+<div class="row-fluid">
```

Comment:
Please see other delete's note the extra are you sure kind of pop up. This should be the same, see https://github.com/stanlyDoge/activemq-artemis/blob/0809b7794e16cd8981209f30f7ec3adf6ce0a114/artemis-hawtio/artemis-plugin/src/main/webapp/plugin/html/deleteAddress.html

Reply:
Hi Michael. After discussion with @andytaylor we decided that divert should not be deletable from web console, so there is no need to have pop-up.

Reply:
If that’s the case please close this pr

Reply:
Eg there is code in this pr that signifies otherwise